src/image/components/runtime.tsx

```diff
@@ -17,10 +19,23 @@ interface IImageSize {
   height: number;
 }
 
-export const Runtime: React.FC<IProps> = ({ authoredState, report }) => {
+export const Runtime: React.FC<IProps> = ({ authoredState, baseContainer, report }) => {
 
   const imageSize = useRef<IImageSize>();
 
+  const creditTextContainer = useRef<HTMLDivElement>(null);
+
+  // We've disabled the autoHeight hook provided by the base app
+  // When the scaling is fitWidth then the image change its size based on the
+  // width of the container, so we should use LARA aspect raio setting. This
+  // way the image's width and height will be limited so the whole thing shows
+  // within the viewport.
+  // When the scaling is originalDimensions then it is best to use autoHeight
+  // because the height of the interactive is fixed based on the image height
+  // the image does not resize in this case so this is perfect for using LARA's
+  // height option
+  useAutoHeight({ container: baseContainer, disabled: authoredState.scaling === "fitWidth" });
```

Comment:
Wouldn't `useAutoHeight` work with the main container (div) defined in this component? 
It could simplify things a bit, as we wouldn't have to pass the ref. But it's not really important.

Another option would be to use `document.body` in `useAutoHeight` and get rid of `container` option (or make it optional). I guess that's what we usually want.

Reply:
I tried using the div defined in the component, but that didn't include the margins that are applied by the base app. So then it wasn't request enough height.

Yeah having useAutoHeight default to the document.body, would make this better. That'd also fix the testing issue since the testing framework wasn't happy initializing a ref inside of the test.